treeherder/perf/fixtures/performance_bug_templates.yaml

```diff
@@ -105,6 +105,10 @@
       Details of the alert can be found in the [alert summary]({{ alertHref }}), including links to graphs and comparisons for each of the affected tests. Please follow our [guide to handling regression bugs](https://wiki.mozilla.org/TestEngineering/Performance/Handling_regression_bugs) and **let us know your plans within 3 business days, or the offending patch(es) will be backed out** in accordance with our [regression policy](https://www.mozilla.org/en-US/about/governance/policies/regressions/).
 
       For more information on performance sheriffing please see our [FAQ](https://wiki.mozilla.org/TestEngineering/Performance/FAQ).
+
+      This is the browsertime test results archive for current revision: {{ resultsLink }}
```

Comment:
Instead of this bottom links, I think we should extend the regression table with an extra column: `Vismetrics` or whatever.
And for each row, we can set `before / after` hyperlinks.

Given that the table is build on frontend, we won't need to update the templates.

Reply:
We've been through this conversation of extending the table: IMO this means altering the current rows to avoid inconsistencies, which might lead to complicated regressions.
But yeah, we can do that also. Not sure if in this sprint.